src/App.jsx

```diff
@@ -14,70 +16,55 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood.length > 0
+          ? `${selectedGood} is selected`
+          : 'No goods selected'}
+        <button
```

Comment:
this button shouldn't be visible if no one good is selected:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/86923089/0cd3d4ff-091e-4c4d-aa22-94f987d97f4d)